pkg/controller/certificates/internal/secretsmanager/keystore.go

```diff
@@ -28,10 +28,9 @@ import (
 	"crypto/x509"
 	"time"
 
-	jks "github.com/pavel-v-chernykh/keystore-go"
-	"software.sslmate.com/src/go-pkcs12"
-
 	"github.com/jetstack/cert-manager/pkg/util/pki"
+	jks "github.com/pavel-v-chernykh/keystore-go/v4"
+	"software.sslmate.com/src/go-pkcs12"
```

Comment:
Please use the [cert-manager import format](https://cert-manager.io/docs/contributing/coding-conventions/#organizing-imports).